package.json

```diff
@@ -20,7 +20,7 @@
     ]
   },
   "engines": {
-    "node": ">=16"
+    "node": ">=14"
```

Comment:
If the CLI won't build in 14, do we really want to make this change? I know the commands run, but isn't the node version here more about what it takes the build the application?

Reply:
We can't run `yarn install` in Node 14 without this change.

Reply:
Fun times!